src/include/uapi/drm_local/amdxdna_accel.h

```diff
@@ -163,6 +163,39 @@ struct amdxdna_ctx_param_config_cu {
 	struct amdxdna_cu_config cu_configs[];
 };
 
+/**
+ * struct uc_info_entry: Holds uc index & buffer size allotment info
+ * @index: uc index
+ *     On aie2ps, uc index is same to column index
+ *     On aie4, uc index is mapped as 0->0_A, 1->0_B, 2->1_A, 3->1_B, 4->2_A, 5->2_B
```

Comment:
This explanation is very hardware specific. Can we make it more generic as this is the interface?